### DISCUSSION TITLE
编译时出现 @umijs\deps\compiled\webpack\4\bundle4.js：64 throw new Error（'Callback was been called.'）

### DISCUSSION THREAD

**Discussion by doYbest** [umijs/umi#8688] [Q&A]
编译时出现 @umijs\deps\compiled\webpack\4\bundle4.js：64 throw new Error（'Callback was been called.'），导致每次终端断开
![image](https://user-images.githubusercontent.com/80409408/180594667-700bd36d-b518-47b9-aceb-59f8a83b6c9a.png)

**Comment by fz6m** [comment]
由于缺乏足够的信息，请提供 [最小重现](https://stackoverflow.com/help/minimal-reproducible-example) 的 demo 帮助排查问题。
